activity_browser/settings.py

```diff
@@ -258,15 +255,15 @@ def db_is_readonly(self, db_name: str) -> bool:
         """Check if given database is read-only, defaults to yes."""
         return self.settings["read-only-databases"].get(db_name, True)
 
-    def get_editable_databases(self):
+    def get_editable_databases(self) -> Generator[str, None, None]:
         """Return list of database names where read-only is false
 
         NOTE: discards the biosphere3 database based on name.
         """
         iterator = self.settings.get("read-only-databases", {}).items()
         return (name for name, ro in iterator if not ro and name != "biosphere3")
```

Comment:
We need to make `biosphere3` configurable, this will change in the future (there won't be just one string). We can either add it to the custom AB settings, or refer to [`bw2data.config.biosphere`](https://github.com/brightway-lca/brightway2-data/blob/wasm/bw2data/configuration.py#L15).

Reply:
Will have a look at the code for the `bw2data` implementation. Would prefer using the brightway standards.